### ISSUE TITLE
Add buttons to download the new installers

### ISSUE THREAD

**Issue by ccordoba12** [spyder-ide/website-spyder#186]
Now that we have the installers, we should promote them in website. So my idea is to change the button

    Download Spyder with Anaconda

for two buttons in a column with the same shape and style, like this:

    Download Spyder for Windows <Windows icon>
    Download Spyder for macOS <macOS icon>

@dalthviz, could you help us with that?

**Comment by CAM-Gerlach** [comment]
@ccordoba12 I can take care of this using the new multi-flowblock feature of Lektor-Icon v2, without having to make any changes in the theme or HTML, just the contents.lr and a whatever CSS tweaks you want.

If you want the buttons in a column (as opposed to putting them side by side at wider screen widths), its trivially simple to achieve: I just added another flowblock to the `contents.lr` for the other button, a few lines of CSS to adjust the size and margins, and voila (we can tweak the style as desired, of course):

![image](https://user-images.githubusercontent.com/17051931/98882324-9f63c980-2451-11eb-96c8-8b085d5e1ba3.png)

If you want them side by side, its almost as simple; we just stick them in a nested flowblock, and with a couple lines of CSS we get something like the following, which even reflows automatically to a single column on mobile (again, we can easily make the buttons bigger, smaller, etc):

![image](https://user-images.githubusercontent.com/17051931/98883853-7d1f7b00-2454-11eb-8fe1-9dfb369e79ac.png)

I shortened the text to avoid awkward and repetitive looking buttons, and I would strongly suggest adding a third option for Linux as to not imply that Spyder doesn't support that platform and thus leave out a large fraction of potential users; we can link to Anaconda, the install guide, or whatever you think is best. 

For the platform icons, we can use Font Awesome to embed them as glyphs in the buttons since it has them for all three platforms; its not built into the theme but we could just copy the CSS we need into our custom stylesheet (and add the appropriate legal text to the footer custom text config option).

Also, we need to remove or replace most of the text that describes Anaconda with something shorter and more appropriate to the new installers. I can propose something simple and we can iterate from there, or I'm happy to work with your ideas if you have something particular in mind.

**Comment by CAM-Gerlach** [comment]
The other PR I was working on to address #183 won't touch the download section (we weren't going to change anything there anyway), so I should be able to finish that in parallel to this one, but let me mock something up here first.

**Comment by dalthviz** [comment]
Sounds good to me @CAM-Gerlach thanks! And as for the alternatives I would say the first one (just one column) looks better. Let me know if you need something from me :+1: and also this change will be worked on #187 ?

**Comment by CAM-Gerlach** [comment]
Thanks @dalthviz 

> also this change will be worked on #187 ?

I thought about it but I ended up making the preliminary changes in a seperate branch, since #187 is ready for review and merge and I didn't want to hold up the simple changes we already agreed on there while we decide on the separate issue of the specifics of the buttons and the text here. I didn't touch the downloads section there, to make sure there shouldn't be any significant merge conflicts with this PR.

**Comment by ccordoba12** [comment]
> And as for the alternatives I would say the first one (just one column) looks better.

I agree with @dalthviz, the buttons looks better in a vertical layout.

Another suggestion (if not too hard to implement) is to add the corresponding operating system icon to the right of to the text. I think that would make the buttons look even better.

**Comment by ccordoba12** [comment]
Also, the paragraphs in the `Download` section need to be rewritten to tell people about our new installers and not Anaconda.

@CAM-Gerlach, could you take care of that too?

**Comment by CAM-Gerlach** [comment]
> Another suggestion (if not too hard to implement) is to add the corresponding operating system icon to the right of to the text. I think that would make the buttons look even better.

Yes indeed, it should be pretty straightforward to do as I discuss above:

> For the platform icons, we can use Font Awesome to embed them as glyphs in the buttons since it has them for all three platforms; its not built into the theme but we could just copy the CSS we need into our custom stylesheet (and add the appropriate legal text to the footer custom text config option).

Instead of manually copying the relevant CSS into ours, I'll just add support for loading multiple custom stylesheets to the theme and add the Font Awesome CSS to the site. Also, icons in buttons, particularly download ones, are conventionally to the left of the text; see e.g. the [Rstudio download page](https://rstudio.com/products/rstudio/download/#download).

> Also, the paragraphs in the `Download` section need to be rewritten to tell people about our new installers and not Anaconda.

Yup, already on it:

> Also, we need to remove or replace most of the text that describes Anaconda with something shorter and more appropriate to the new installers. I can propose something simple and we can iterate from there, or I'm happy to work with your ideas if you have something particular in mind.

@ccordoba12 What do you think about

> I would strongly suggest adding a third option for Linux as to not imply that Spyder doesn't support that platform and thus leave out a large fraction of potential users; we can link to Anaconda, the install guide, or whatever you think is best.

Otherwise, it looks to users like Spyder doesn't support Linux at all (even if we mention it in the text, they may not spend their time reading it if they just see download buttons for Windows and macOS).

**Comment by ccordoba12** [comment]
> Otherwise, it looks to users like Spyder doesn't support Linux at all (even if we mention it in the text, they may not spend their time reading it if they just see download buttons for Windows and macOS).

Agreed. The button should point to Anaconda.
